integration_tests/build.gradle.kts

```diff
@@ -38,19 +38,45 @@ tasks.test {
     }
 }
 
-tasks.register<Test>("integrationTests") {
+tasks.register<Test>("integrationTestsAndroid") {
     group = "Verification"
-    description = "Runs flank integration tests"
+    description = "Runs flank integration tests -- Android"
     filter {
         includeTestsMatching("*IT")
     }
+    useJUnit {
+        includeCategories = setOf("integration.config.AndroidTest")
+        excludeCategories = setOf("integration.config.IosTest")
+    }
     testLogging {
         events("skipped", "failed")
         exceptionFormat = TestExceptionFormat.FULL
     }
     maxParallelForks = Runtime.getRuntime().availableProcessors() / 2
```

Comment:
Is this necessary? Lets remove this and see if it speeds it up.

Reply:
i.e.:
```suggestion
    maxParallelForks = Runtime.getRuntime().availableProcessors()
```

Reply:
Why not, let's give it a try 👍🏻